test/projects/test_binary_search.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+
+from runner import ProjectType
+from glotter import project_test, project_fixture
+from test.utilities import clean_list
+
+invalid_permutations = (
+    'description,in_params,expected', [
+        (
+            'no input',
+            None,
+            'Usage: please provide two lists in the format "1, 2, 3, 4, 5"'
+        ), (
+            'empty input',
+            '""',
+            'Usage: please provide two lists in the format "1, 2, 3, 4, 5"'
+        ), (
+            'missing input',
+            '"25 15 10 5"',
+            'Usage: please provide two lists in the format "1, 2, 3, 4, 5"'
+        )
+    ]
+)
+
+valid_permutations = (
+    'description,in_params,expected', [
+        (
+            'sample input same length',
+            '"1, 4, 5, 3, 15, 6" "1, 7, 4, 5, 11, 6"',
+            '1, 4, 5, 6'
+        ), (
+            'sample input different length',
+            '"1, 4, 8, 6, 9, 3, 15, 11, 6" "1, 7, 4, 5, 8, 11, 6"',
+            '1, 4, 8, 11, 6'
+        )
+    ]
+)
+
+
+@project_fixture(ProjectType.LCS.key)
+def lcs(request):
```

Comment:
I think you forgot to rename lcs to binary search

Reply:
I haven't even added tests yet. Haha Just copied another example for myself.

Reply:
Bad timing. Just happened to be looking at my phone right when you created this. Haha.

Reply:
Haha no sweat! I appreciate the review.